mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -177,10 +177,10 @@ object Tool extends LazyLogging {
   }
 
   private def setCoreOptions(executor: PassChainExecutor, cmd: AbstractCheckerCmd): Unit = {
-    logger.info(
-        "Checker options: filename=%s, init=%s, next=%s, inv=%s"
-          .format(cmd.file, cmd.init, cmd.next, cmd.inv)
-    )
+    logger.info {
+      val environment = if (cmd.env != "") s"(${cmd.env})" else ""
+      s"Checker options: ${environment} ${cmd.name} ${cmd.invocation}"
```

Comment:
Can I nitpick? This will print two consecutive blanks on an empty `cmd.env` 😬
Feel free to mark this resolved without a change.